README.md

```diff
@@ -1,40 +1,60 @@
-# Arduino Library for network connections management
-
-Library for handling and managing network connections by providing keepAlive functionalities and reconnection attempts. Initially for WiFi, GSM and NB, its goal is to be extended to more networking layers over diverse hardware (Ethernet, BlueTooth et al.)
-
-
-## Usage
-
-## Credits
-
-## Website
-
-# License
-
-This library is free software; you can redistribute it and/or
-modify it under the terms of the GNU Lesser General Public
-License as published by the Free Software Foundation; either
-version 2.1 of the License, or (at your option) any later version.
-
-This library is distributed in the hope that it will be useful,
-but WITHOUT ANY WARRANTY; without even the implied warranty of
-MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
-Lesser General Public License for more details.
-
-You should have received a copy of the GNU Lesser General Public
-License along with this library; if not, write to the Free Software
-Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
-
-
-### ConnectionHandler
-
-**Connection Handler** is configured via a series of compiler directives, including the correct implementation of the class based on which board is selected.
-
-### How to use it
-- Instantiate the class with `ConnectionHandler conHandler(SECRET_SSID, SECRET_PASS);` if you are using a WiFi board, otherwise replace **WiFi** with **GSM** or **NB** or any future implementation minding to carefully pass the init parameters to the constructor (i.e.: for the GSM you'll need to pass PIN, APN, login, password).
-
-- The `update()` method does all the work. It uses a finite state machine and is responsible for connection and reconnection to a network. The method is designed to be non-blocking by using time (milliseconds) to perform its tasks.
-
-- `&getClient()` returns a reference to an instance of the `Client` class used to connect to the network.
-
-- `getStatus()` returns the network connection status. The different states are defined in an `enum`
+Arduino Library for network connections management
+==================================================
+
+Library for handling and managing network connections by providing keep-alive functionality and automatic reconnection in case of connection-loss. It supports the following boards:
+* **WiFi**: `MKR 1000`, `MKR WiFi 1010`, `ESP8266`
+* **GSM**: `MKR GSM 1400`
+* **5G**: `MKR NB 1500`
+* **LoRa**: `MKR WAN 1300/1310`
```

Comment:
```suggestion
* **WiFi**: [`MKR 1000`](https://store.arduino.cc/arduino-mkr1000-wifi), [`MKR WiFi 1010`](https://store.arduino.cc/arduino-mkr-wifi-1010), `ESP8266`
* **GSM**: [`MKR GSM 1400`](https://store.arduino.cc/arduino-mkr-gsm-1400-1415)
* **5G**: [`MKR NB 1500`](https://store.arduino.cc/arduino-mkr-nb-1500-1413)
* **LoRa**: [`MKR WAN 1300/1310`](https://store.arduino.cc/mkr-wan-1310)
```
Just a thought, to make it easy for people to find info on these boards and give a little love to the Arduino Store.